### PR TITLE
Feature/sponsors resources

### PR DESCRIPTION
**Pull request by StrangeWill**

### CODE REVIEW

File: _includes/resource.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div class="resource">
+    <div>
+        <a href="{{item.website}}" titl="{{item.name}}" class="resource--icon">
+            <img src="/images/{{item.type}}/{{item.icon}}" alt="{{item.name}}"/>
+        </a>
+    </div>
+    <div>
+        <a href="{{item.website}}" title="{{item.name}}’s website" class="resource--title">
```
Comment (t-richards) [inline _includes/resource.html:8]: Is this intentionally a fancy quote?
```suggestion
        <a href="{{item.website}}" title="{{item.name}}'s website" class="resource--title">
```

File: _sass/_resources.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.resource {
+  clear: both;
+  //border: solid 1px #ccc;
```
Comment (t-richards) [inline _sass/_resources.scss:3]: ✂️

File: _sass/_resources.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  .resource--icon {
+    img {
+      width: 100px;
+    }
+  }
+
+  .resource--about {
+    //text-align: left;
+  }
```
Comment (t-richards) [inline _sass/_resources.scss:25]: ✂️

File: images/events/chadev.svg [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (t-richards) [inline images/events/chadev.svg:2]: Highly recommend running this (really all images for that matter) through something like https://imageoptim.com/ to remove unnecessary cruft such as this comment 😎

File: _includes/resource.html (hunk 1 above)
Comment (samlachance) [reply _includes/resource.html:8]: can't afford a mac huh?

File: images/events/chadev.svg (hunk 4 above)
Comment (StrangeWill) [reply images/events/chadev.svg:2]: Didn't even think of things like this in SVGs. 👍

File: _includes/resource.html (hunk 1 above)
Comment (StrangeWill) [reply _includes/resource.html:8]: I don't even use one so I have no clue how that got in there.

**Pull request closed without merging**